bk-config/README.md

```diff
@@ -0,0 +1,43 @@
+
+This container 
+
+
+```bash
+
+docker build -t sagecontinuum/bk-config .
+
+```
+
+
+
+# Create CA and beekeeper key-pairs and certificates and registration key (NOT registration cert)
+
+```bash
+docker run -ti --rm --network beekeeper_default --name bk-config -v beekeeper-config_bk-secrets:/usr/lib/sage/ sagecontinuum/bk-config
+
+
+init-keys.sh
+```
+
+Afterwards you can start beekeeper.
+
+
+# create client files (e.g. for a node) 
+
+```bash
+./create_client_files.sh
+```
+
```

Comment:
An example might be valid here with proper parameters to the `./create_client_files.sh` script

Reply:
done